LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2017
```

Comment:
I think we should have company name here. 

@cyril-sf  @doermann

Reply:
It's MIT, and open source. It'll be a pain to remember this because this is the default file generated by Ember for new addons.